clang/runtime/dpct-rt/include/dpct/group_utils.hpp

```diff
@@ -0,0 +1,541 @@
+//==---- group_utils.hpp ------------------*- C++ -*--------------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------===//

```

Comment:
```suggestion
//===------------------------------------------------------------------===//
```
Align with top line of header